api.ts

```diff
@@ -0,0 +1,28 @@
+// api.ts
+
+import axios from 'axios';
+
+const API_URL = 'https://keepitfresh-d0evbuaud4afdqd7.australiaeast-01.azurewebsites.net/api/shopping-list/items/';
```

Comment:
Thanks Jordan, it's fine for first iteration but going forward we can externized this to a config, so no code rebuild required when API url changed